packages/nextjs/app/builders/page.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import MembersList from "./_components/MembersList";
+import fs from "fs/promises";
+import path from "path";
+
+// Getting teh buildres dir. data
```

Comment:
Typo: "teh buildres"

Reply:
Fixed the typo, from "teh buildres" to "the buildres"